docs/continuous-delivery/low-code-solutions/low-code-solutions.md

```diff
@@ -0,0 +1,37 @@
+# Continuous delivery on low code solutions
+
+Low code, no code platforms have taken a spot in a wide variety of Business Solutions involving process automation, AI models, Bots, Business Applications and Business Intelligence. The scenarios enabled by these platforms are constantly evolving and opening a spot for productive roles. This has been exactly the reason why bringing more professional tools to their development has became necessary such as controlled and automated delivery.
+
+In the case of Power Platform products, the adoption of a CI/CD process may increase the development complexity to a solution oriented to [Citizen Developers](https://www.gartner.com/en/information-technology/glossary/citizen-developer) but, on the other hand, it makes the development process more scalable and capable of dealing with new features and bug corrections in a faster way.
```

Comment:
I'd put this sentence in a more positive way. Something like:

Although the adoption of a CI/CD process in a Power Platform development environment may seem to increase the complexity to a solution oriented to [Citizen Developers](https://www.gartner.com/en/information-technology/glossary/citizen-developer) it is important to make the development process more scalable and capable of dealing with new features and bug corrections in a faster way.